src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml.cs

```diff
@@ -81,6 +101,13 @@ private void RegisterButtonHandlers()
 
         private void RegisterViewEventHandlers()
         {
+            //var binding = new Binding("ActiveGeometry")
+            //{
+            //    Source = ViewModel,
+            //    Mode = BindingMode.OneWay,
+            //};
+            //watch_view.SetBinding(AttachedProperties.ActiveGeometryProperty, binding);
+
```

Comment:
NIT: We could clean up this commented out code.

Reply:
Will remove